homeassistant/components/media_player/plex.py

```diff
@@ -62,6 +107,37 @@ def config_from_file(filename, config=None):
 
 def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """Setup the Plex platform."""
+    # optional parameters
+    optional_config = {}
+    optional_config["entity_namespace"] = config.get("entity_namespace")
```

Comment:
This pretty much seems a mapping of config to a new dict called `optional_config`. Why not just use `config` ?